contracts/suilend/sources/lending_market.move

```diff
@@ -54,7 +56,7 @@ module suilend::lending_market {
 
         // window duration is in seconds
         rate_limiter: RateLimiter,
-        fee_receiver: address,
+        fee_receiver: address, // deprecated
```

Comment:
i guess we can use this for something else later lol? like as long as it's something that just needs an address?